app/views/shared/sidebar/_sidebar.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class='sidebar'>
+  <div class='text-white w-3/4 self-center my-10'>
+    <svg xmlns='http://www.w3.org/2000/svg' id='Layer_1' x='0' y='0' version='1.1' viewBox='0 0 369.9 37.5' xml:space='preserve' fill='currentColor'>
```

Comment:
We should store this as an asset and then show it in the view using the Rails helper